_rules/role-attribute-valid-value-674b10.md

```diff
@@ -112,44 +117,48 @@ Element with role attribute that is not empty (""), neither a valid `role` value
 
```

Comment:
Inapplicable examples have been moved around to mimic the order of conditions in the applicability.
They haven't change, despite what Github believes 😉